core/src/main/scala/com/intenthq/pucket/Pucket.scala

```diff
@@ -201,6 +290,8 @@ trait PucketCompanion {
 object Pucket {
   val defaultBlockSize = 50 * 1024 * 1024
   val extension = ".parquet"
+  val defaultCreationAttempts = 1
+  val defaultRetryIntervalMs = 1000
```

Comment:
This is sort of surprisingly high; even in a production environment I would have thought you'd probably still get away with a smaller value here - maybe like 100 or 250ms at most?